src/components/LiveStatusPoller.js

```diff
@@ -1,20 +1,41 @@
 import React from 'react'
 import { connect } from 'react-redux'
 import { fetchLiveStatus } from '../actions'
-const { func } = React.PropTypes
+const { func, string } = React.PropTypes
 
 @connect()
 export default class LiveStatusPoller extends React.Component {
 
   static propTypes = {
-    dispatch: func
+    dispatch: func,
+    communityId: string
   }
 
-  componentDidMount () {
+  setPollInterval (communityId) {
     let { dispatch } = this.props
-    dispatch(fetchLiveStatus())
-    setInterval(() => dispatch(fetchLiveStatus()),
-      60 * 1000)
+
+    if (this.pollInterval) {
+      clearInterval(this.pollInterval)
+    }
+
+    this.pollInterval = setInterval(() => dispatch(fetchLiveStatus(communityId)),
+      6 * 1000)
+  }
+
+  componentDidMount () {
+    let { dispatch, communityId } = this.props
+    dispatch(fetchLiveStatus(communityId))
```

Comment:
can we defer this initial fetch for, say, 10 seconds, just to reduce the amount of work that happens during mounting?